console/console-init/ui/src/Components/ConnectionDetail/ConnectionDetail.css

```diff
@@ -0,0 +1,44 @@
+.l_split_m_gutter_MarginRight {
+  font-size: 1em;
+  border-right: 2px solid lightgrey;
+  /* margin-left: 10px; */
+  margin-right: 20px;
+}
+.l_split_m_MarginRight {
+  font-size: 1em;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  padding-right: 10px;
+}
+.l_split_m_dropdown {
+  padding-top: 10px;
+  padding-bottom: 10px;
+  margin-left: 20px;
+  font-size: 1em;
+  color: rgb(0, 102, 204);
+}
+.l_split_m_height {
+  font-size: 2em;
+}
+.pf-c-dropdown__menu-item {
```

Comment:
This is still an issue, we can't style a `pf` class directly without namespacing it to tie it to the `ConnectionDetail` component.

In general this `l` namespace is too generic. Maybe `ConnectionDetail` would be a better candidate. Or we could do CSS in JS using `@patternfly/react-styles`